apps/web/src/pages/Profile/EmploymentEquityPage/components/EmploymentEquityForm/EmploymentEquityForm.tsx

```diff
@@ -11,6 +11,7 @@ import ProfileFormWrapper, {
   ProfileFormFooter,
 } from "~/components/ProfileFormWrapper/ProfileFormWrapper";
 
+import { useSearchParams } from "react-router-dom";
```

Comment:
nitpicky but, can we ensure third party imports are at the top 😅

Reply:
see: [Move third party import to top](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5605/commits/219f019d33eb2a0545bdb5e7940c282e47300e9d)